openlibrary/templates/books/custom_carousel.html

```diff
@@ -50,7 +50,7 @@
       </a>
     </div>
     $if cta:
-      <div class="book-cta"><a class="btn primary $modifier" href="$cta_url" data-ol-link-track="$key" title="$cta $book.title"
+      <div class="book-cta"><a class="cta-btn cta-btn--available $modifier" href="$cta_url" data-ol-link-track="$key" title="$cta $book.title"
```

Comment:
standardizes cta buttons to use the same classes on a page

Reply:
What's $modifier used for? (Cannot see rest of file on mobile device).